src/osd/ReplicatedPG.cc

```diff
@@ -5977,6 +5979,23 @@ int ReplicatedPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
       }
       break;
 
+    case CEPH_OSD_OP_ASSERT_INTERVAL:
+      ++ctx->num_read;
+      {
+	epoch_t epoch = op.assert_interval.epoch;
+	epoch_t pg_epoch = get_epoch();
```

Comment:
Hmm, he's doing that using pg_has_reset_since below.  Here, he's checking to make sure that the pg has a new enough map to handle this message.  I *think* that this must always pass since the MOSDOp message epoch would have at least as recent an epoch as the scrub epoch, and we would have delayed the message if the pg didn't have a map epoch at least that recent.